src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -1413,6 +1416,54 @@ class spell_dru_omen_of_clarity_restoration : public AuraScript
     }
 };
 
+// 383410 - Celestial Alignment
+// 390414 - Incarnation: Chosen of Elune
+class spell_dru_orbital_strike : public SpellScript
+{
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_DRUID_ORBITAL_STRIKE_TALENT, SPELL_DRUID_ORBITAL_STRIKE_DAMAGE });
+    }
+
+    bool Load() override
+    {
+        return GetCaster()->HasAura(SPELL_DRUID_ORBITAL_STRIKE_TALENT);
+    }
+
+    void HandleDamage(SpellEffIndex /*effIndex*/)
+    {
+        GetCaster()->CastSpell(GetHitUnit(), SPELL_DRUID_ORBITAL_STRIKE_DAMAGE, CastSpellExtraArgs()
```

Comment:
This script is very wrong

* 361237 is aoe damage
* the purpose of this dummy effect is only to make client send a spell destination in cast spell packet
* 383410/390414 trigger 390381 which summons a moving areatrigger (starting from caster position to the destination selected for this spell)
* areatrigger casts 361237 (the damage spell) every 6 yards (movespeed should be ~45 according to my calculations)
* areatrigger has default duration of 3 seconds that must be shortened to total travel time